applications/vessel-history/package.json

```diff
@@ -0,0 +1,144 @@
+    "i18next-http-backend": "^1.0.21",
+    "intersection-observer": "^0.11.0",
+    "lodash": "^4.17.20",
+    "luxon": "^1.25.0",
+    "qs": "^6.9.4",
+    "react": "^17.0.1",
+    "react-debounce-input": "^3.2.3",
+    "react-dom": "^17.0.1",
+    "react-i18next": "^11.7.3",
+    "react-redux": "^7.2.2",
+    "react-scripts": "4.0.1",
+    "react-sticky-el": "^2.0.5",
+    "redux": "^4.0.5",
+    "redux-first-router": "^2.1.5",
+    "typescript": "^4.1.2",
+    "viewport-mercator-project": "^7.0.1"
+  },
+  "devDependencies": {
+    "@globalfishingwatch/linting": "^1.5.4",
+    "@testing-library/jest-dom": "^5.11.5",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/classnames": "^2.2.10",
+    "@types/file-saver": "^2.0.1",
+    "@types/jest": "^26.0.14",
+    "@types/lodash": "^4.14.161",
+    "@types/luxon": "^1.25.0",
+    "@types/node": "^14.11.2",
+    "@types/qs": "^6.9.5",
+    "@types/react": "^16.9.56",
+    "@types/react-dom": "^16.9.9",
+    "@types/react-map-gl": "^5.2.7",
+    "@types/react-redux": "^7.1.9",
+    "@types/react-sticky-el": "^1.0.2",
+    "@types/redux-first-router": "^2.1.5",
+    "@types/viewport-mercator-project": "^6.1.1",
+    "@typescript-eslint/eslint-plugin": "^4.6.1",
+    "@typescript-eslint/parser": "^4.6.1",
+    "@typescript-eslint/scope-manager": "^4.6.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "7.12.1",
+    "eslint-config-prettier": "6.15.0",
+    "eslint-config-react-app": "6.0.0",
+    "eslint-import-resolver-typescript": "2.3.0",
+    "eslint-plugin-flowtype": "5.2.0",
+    "eslint-plugin-import": "2.21.1",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "3.1.4",
+    "eslint-plugin-react": "7.21.5",
+    "eslint-plugin-react-hooks": "4.2.0",
+    "husky": "^4.3.0",
+    "i18next-parser": "^3.3.0",
+    "imagemin-lint-staged": "^0.4.0",
+    "lint-staged": "^10.4.0",
+    "prettier": "^2.1.2",
+    "react-test-renderer": "^17.0.1",
+    "stylelint": "^13.7.1",
+    "stylelint-config-css-modules": "^2.2.0",
+    "stylelint-config-prettier": "^8.0.2",
+    "stylelint-config-standard": "^20.0.0"
+  },
+  "eslintConfig": {
+    "extends": [
+      "@globalfishingwatch/linting/typescript"
+    ],
+    "ignorePatterns": "public/"
+  },
+  "prettier": "@globalfishingwatch/linting/prettier",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": [
+      "eslint -c ./.eslintrc.js --fix"
+    ],
+    "*.css": [
+      "stylelint --fix",
+      "stylelint --syntax=scss --fix"
+    ],
+    "*.scss": [
+      "prettier --write",
+      "stylelint --syntax=scss --fix"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "jest": {
```

Comment:
Is there a way to centralice the configuration ? The same that we have with tsconfig and rollup config

Reply:
Yes, there's an approach that lets define a jest.config.base.js in the root folder of the monorepo and then include it in the jest.config.js of each project. I tried to setup it that way but it did not work. We could give it another try later or talk about it.